cmd/init.go

```diff
@@ -74,11 +75,11 @@ setup the local dev environment.`,
 
 	initCmd.PersistentFlags().BoolVar(&config.overwrite, "overwrite", false, "Download and extract the template project, overwriting existing files.  This option is not intended to be used in Appsody project directories.")
 	initCmd.PersistentFlags().BoolVar(&config.noTemplate, "no-template", false, "Only create the .appsody-config.yaml file. Do not unzip the template project. [Deprecated]")
+	initCmd.PersistentFlags().StringVar(&config.projectName, "project-name", "", "Project Name for Kubernetes Service")
```

Comment:
It would be nice to set the default here, so the `appsody help init` command will display the proper default. 
See `run` and `extract` commands for examples of how to do that.

Reply:
If I set the default value of projectName in the way you are setting in extract.go for extractContainerName then the default value is always set to my-project because we are setting projectName based on current directory name and we can't get the directory name until we init the project because getProjectDir() function gives you the dir name only if rootConfig has the value and it has the value when project is initialized.

Reply:
The `defaultExtractContainerName()` method works today, such that `appsody help extract` will lists the correct default name based on the projectDir, not "my-project". 

Are you saying this PR breaks that?

I don't see a reason why you can’t run the same logic (reusing code) to calculate the default `init --project-name` for the help text in a similar way to extract's name. The `config.ProjectDir` should be set by then. 

I feel pretty strongly about this because it impacts the user’s first impression of appsody.

Reply:
I have changed it. Now I'm also getting the default name similar to the way you are getting default Name. You are also getting name always "my-project" but you are just appending "-extract" at the end of "my-project".